### DISCUSSION TITLE
Creating an order programmatically - TAX reset

### DISCUSSION THREAD

**Discussion by osher-retailin** [saleor/saleor#8460] [Q&A] [answered]
**What I'm trying to achieve**
Creating an order through the "draftOrderCreate" and "draftOrderComplete".

**What's the problem**
"Taxes" field of the order created via "draftOrderCreate" is being reset when draft order is complete and order is created.
Another problem, the is not being recalculated when quantity or price of an order item is changed.

I'm working on Saleor version **3.0.0-a.16**.

Best regards for the helpers.

**Comment by karolkielecki** [comment, marked as answer]
@osher-retailin Hi there. Please use the latest beta version. The latest version contains all the improvements and bug fixes. If you still experience this issue in the beta version, please let me know.

**Reply by osher-retailin** [reply]
Ok, THNX for the reply.
